console/billing/api-cli/retrieve-monthly-consumption.mdx

```diff
@@ -0,0 +1,101 @@
+---
+meta:
+  title: Retrieving your monthly consumption via the Scaleway API
+  description: This page explains how to retrieve your monthly consumption via the Scaleway API
+content:
+  h1: Retrieving your monthly consumption via the Scaleway API
+  paragraph: This page explains how to retrieve your monthly consumption via the Scaleway API
+tags: billing consumption report api export
+dates:
+  validation: 2024-03-26
+  posted: 2024-03-26
+categories:
+  - billing
+---
+
+Follow the procedure below to download your monthly consumption using the Scaleway [Billing API](https://www.scaleway.com/en/developers/api/billing/#path-consumption-get-monthly-consumption) and `curl`.
```

Comment:
```suggestion
Follow the procedure below to download your monthly consumption using the Scaleway [Billing API](https://www.scaleway.com/en/developers/api/billing/#path-consumption-get-monthly-consumption).
```

Reply:
Removed "and `curl`" because there is no procedure that shows how to download it in the doc + it's a requirement to have downloaded it before starting.

Reply:
I wrote "using the Scaleway API and curl" bc you can use the apis with different tools. in this tuto we use curl. I'll keep in the text

Reply:
Ah sorry I thought "and curl" went with "download"